src/spark_manager.py

```diff
@@ -165,12 +166,11 @@ def create_cluster(
         )
         master_ui_url = f"http://{self.master_name}.{self.namespace}:{self.DEFAULT_MASTER_WEBUI_PORT}"
 
-        return {
-            "cluster_id": self.cluster_id,
-            "master_url": master_url,
-            "master_ui_url": master_ui_url,
-            "status": "creating",
```

Comment:
Status isn't needed any more?

Reply:
status is always going to be `creating`. I don't think it's necessary.

Reply:
:+1: